client/bot-community.py

```diff
@@ -36,6 +34,82 @@
 NUMBER_OF_LOCATIONS = 100
 MAX_POPULATION = 12 * NUMBER_OF_LOCATIONS
 
+
+@click.group()
+@click.option('--client', default='../target/release/encointer-client-notee', help='the client to communicate with the chain')
+@click.option('--port', default='9944', help='port for the client to communicate with chain')
+@click.option('-l', '--ipfs_local', is_flag=True, help='if set, local ipfs node is used')
+@click.option('--node_url', default=None, help='if set, remote chain is used with port 443, no need to manually set port, it will be ignored')
```

Comment:
We should discuss this. I left node_url a string instead of bool because I wondered, if we will want to add other url's. If yes, i can just make it directly generic and not hardvode the string.

Reply:
@brenzi do you wish the remote node url to be hardcoded or configurable?

Reply:
Now it makes sense to hard-code them, as I can only think of 4 encointer chains that will be run long-term remotely.

* gesell-kusama
* gesell-polkadot
* cantillon-kusama
* cantillon-polkadot

Short term we might have gesell-/cantillon-westen, but I doubt that these scripts should support that.

Reply:
do gesell-dot, cant ksm & cant dot exist? I don't see them on polkadot-js. if yes, what are the ws adresses?

Reply:
Ah sorry, I forgot the two obvious ones: gesell and cantillon standalone. These are the only ones that already run.

The ones from my previous comment don't exist yet.

Reply:
is for cantillon the address: 
wss://cantillon.encointer.org
?

Reply:
It would be, but it does not seem to be running, nor is it supported currently. Hence, I would not add it for now.